Lumafly/ViewModels/InfoViewModel.cs

```diff
@@ -139,8 +138,15 @@ static bool IsHollowKnight(Process p) => (
 
         if (hkExeFolder is null) throw new Exception("Hollow Knight executable not found");
         string exePath = hkExeFolder.FullName;
+        
+        // check if path contains steam_api64.dll
+        var x86_64Folder = managedParent?
+            .GetDirectories().FirstOrDefault(d => d.Name == "Plugins")?
+            .GetDirectories().FirstOrDefault(d => d.Name == "x86_64");
+        
+        var isSteam = x86_64Folder?.GetFiles("steam_api64.dll").Any() ?? false;
```

Comment:
This entire section can be replaced:
```cs
var isSteam = File.Exists(Path.Combine(
    managedParent.Path,
    "Plugins",
    "x86_64",
    "steam_api64.dll"
));
```
This avoids a fair amount of IO calls and allocations.